src/com/android/launcher3/Utilities.java

```diff
@@ -159,7 +156,7 @@
      */
     public static final boolean IS_DEBUG_DEVICE =
             Build.TYPE.toLowerCase(Locale.ROOT).contains("debug") ||
-            Build.TYPE.toLowerCase(Locale.ROOT).equals("eng");
+                    Build.TYPE.toLowerCase(Locale.ROOT).equals("eng");
```

Comment:
Can we get rid of all formatting changes in unrelated code in this PR?

Reply:
OK! What I remember is that I actually copy & pasted the official Lawnchair Utilities.java here but Android Studio decided to correct the changes all the same

Reply:
Well you can just discard them fro git

Reply:
> Well you can just discard them from git

Which is what I would have done, except that it’s broken in AS 3.5

Anyway I ended up doing it from the command-line